app/models/concerns/userable.rb

```diff
@@ -33,5 +34,6 @@ def self.validate_password
     validate_user_name(:last_name)
     validate_email
     validate_password
+    validate_field_presence(:status)
```

Comment:
We're discussing that a method `validate_field_presenc` is not necessary and we have a `default value` so if it's empty or nil. we catch `visitor`

Reply:
rmv